frontend/src/services/proprietaryService.ts

```diff
@@ -41,8 +38,43 @@ export function useProprietaryService(
     const eliVal = toValue(eli)
     if (!eliVal || !dateAsString.value) return INVALID_URL
 
-    return `/norms/${toValue(eli)}/proprietary/${dateAsString.value}`
+    return `/norms/${eliVal}/proprietary/${dateAsString.value}`
   })
 
-  return useApiFetch<Proprietary>(url, { ...fetchOptions }).json()
+  return useApiFetch<Proprietary>(url, fetchOptions)
+}
+
+/**
+ * Convenience shorthand for `useProprietaryService` that sets the correct
+ * configuration for getting JSON data.
+ *
+ * @param eli ELI of the norm
+ * @param options Optional additional filters and queries
+ * @param [fetchOptions={}] Optional configuration for fetch behavior
+ * @returns Reactive fetch wrapper
+ */
+export const useGetProprietary: typeof useProprietaryService = (
+  eli,
+  options,
+  fetchOptions,
+) => useProprietaryService(eli, options, fetchOptions).json()
+
+/**
+ * Convenience shorthand for `useProprietaryService` that sets the correct
+ * configuration for putting JSON data.
+ *
+ * @param eli ELI of the norm
+ * @param options Optional additional filters and queries
+ * @param [fetchOptions={}] Optional configuration for fetch behavior
+ * @returns Reactive fetch wrapper
+ */
+export function usePutProprietary(
+  updateData: MaybeRefOrGetter<Proprietary | null>,
+  eli: Parameters<typeof useProprietaryService>["0"],
+  options: Parameters<typeof useProprietaryService>["1"],
+  fetchOptions: Parameters<typeof useProprietaryService>["2"],
```

Comment:
for a put the current options do not have an effect and the stuff that needs to be passed to `fetchOptions` should be handled here (atleast in my understanding). So do we really need/want those parameters here? I would try to keep the parameters for these methods as simple as possible and only extend them if we actually need to support more stuff

Reply:
Agree, I'll change it

Reply:
I checked again and the options do have an effect and this is used (see `AmendingLawAffectedDocumentRahmenEditor.vue` for an example). So I'm keeping it as it is